CHANGELOG.rst

```diff
@@ -10,6 +10,8 @@ Unreleased
 * Remove debug print from apphook_reload
 * Enforce use of coverage > 4 for python 3.8 support
 * Fixed 66622 bad Title.path in multilingual sites when parent slug is created or modified
+* Fixed 6973 bag with unexpected behavior get_page_from_request
```

Comment:
This should use double back-ticks for inline code, e.g. ``, at either end of get_page_from_request

Docs here; https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup